staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -52,5 +52,6 @@ const (
 	// Service is Headless. The existence of this label can be used further by other
 	// controllers and kube-proxy to check if the Endpoint objects should be replicated when
 	// using Headless Services
-	IsHeadlessService = "service.kubernetes.io/headless"
+	IsHeadlessService  = "service.kubernetes.io/headless"
+	LabelNamespaceName = "kubernetes.io/metadata.name"
```

Comment:
add a newline and godoc. Suggest `LabelMetadataName`, and documenting the in-tree use for labeling namespaces

Reply:
done